content/docs/how-we-work/task-estimation/estimation-process.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Estimation Process
+weight: 5
+description: It’s essential to discuss the estimation process during onboarding to understand how it’s handled for each project. Different projects might have varying estimation methods.
+---
+
+# Estimation Process
+
+It’s essential to discuss the estimation process during onboarding to understand how it’s handled for each project. Different projects might have varying estimation methods. Below are some commonly encountered practices:
+
+## Grooming
+
+We ensure that the necessary grooming is done, and the required details are added to the Implementation details Jira field. The Technical Architect (TA) can be consulted if guidance on the approach is needed. Adding implementation details helps the client understand where the hours will be utilized.
+
+## Estimation Templates
+
+Use the [master estimate sheet template](https://docs.google.com/spreadsheets/d/1wXAFXsSbiQfaMSCMGr-7GvWHYh2O9fc5mhUOQgJsos8/edit?gid=683107236#gid=683107236) and follow the instructions given in the sheet. The team can raise a request or suggest feedback or invite a call to discuss any challenges, which will help revise the estimation sheet iteratively. Add estimates in internal and bug tickets as well and get those reviewed by the TA. Add a link and a screenshot of the estimation sheet on the respective ticket. The screenshot is important as it will help us identify any changes done later to the estimations. We also use the [CDM ticket estimation template](https://docs.google.com/spreadsheets/d/1NExM6FlLe0kWwhFNPp4RBi8E1D7iixRQHPVriP22B1Y/edit?gid=0#gid=0) only for ticket-level estimation.
+
+## Sprint Planning Meetings
+
+Estimation is often conducted during sprint planning meetings. This allows the team to discuss and refine estimates based on the latest information and priorities.
```

Comment:
@Yemaneberhan-Lemma I think we can remove this as we are not conducting Sprint planning meetings per se for estimations and rather do it async and collaborate if needed.